test/integration/targets/hwc_vpc_route/tasks/main.yml

```diff
@@ -0,0 +1,117 @@
+---
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file at
+#     https://www.github.com/huaweicloud/magic-modules
+#
+# ----------------------------------------------------------------------------
+# Pre-test setup
+- name: create a vpc
+  hwc_network_vpc:
+    cidr: "192.168.0.0/16"
+    name: "ansible_network_vpc_test_local"
+    state: present
+  register: vpc1
+- name: create a vpc
+  hwc_network_vpc:
+    cidr: "192.168.0.0/16"
+    name: "ansible_network_vpc_test_peering"
+    state: present
+  register: vpc2
+- name: create a peering connect
+  hwc_vpc_peering_connect:
```

Comment:
This module is not part of Ansible?

Reply:
It's in https://github.com/ansible/ansible/pull/60190